src/metagame.ts

```diff
@@ -1,22 +1,8 @@
+import type { StatsTable } from "@pkmn/dex";
 import { getMean, getStdev } from "./math";
 
-interface Stats {
-  hp: number;
-  atk: number;
-  def: number;
-  spa: number;
-  spd: number;
-  spe: number;
-}
-
-interface EffectiveStats {
-  hp: number;
-  atk: number;
-  def: number;
-  spa: number;
-  spd: number;
-  spe: number;
-}
+type Stats = StatsTable;
+type EffectiveStats = StatsTable;
```

Comment:
See my comments on a past commit about how since nominal typing does not exist in Typescript (or well, you can fudge it a little bit with 'brands'), because these types have the same structure, attempting to differentiate `Stats` and `EffectiveStats` doesnt really accomplish very much (`type` is just creating an alias and won't actually provide any safety guarantees).

Reply:
👍 I'll make these types more useful in the next PR. :)